packages/bundler/src/modules/BundleManager.ts

```diff
@@ -257,19 +267,21 @@ export class BundleManager {
         break
       }
 
-      if (paymaster != null) {
+      if (paymaster != null && isAddress(paymaster) && paymaster === AddressZero) {
```

Comment:
shouldn't it be paymaster != AddressZero ?

Reply:
Indeed. Fixed.